rocketpy/EnvironmentAnalysis.py

```diff
@@ -966,26 +967,68 @@ def animate_wind_gust_distribution_over_average_day(self):
         """Animation of how the wind gust distribution varies throughout the day."""
         ...
 
-    # TODO: Create tests
-    def animate_wind_profile_over_average_day(self):
-        """Animation of how wind profile evolves throughout an average day."""
-        # Gather animation data
-        altitude_list = np.linspace(1495.155, 12000, 100)  # TODO: parametrize
+    # TODO: Create test
+    def process_wind_profile_over_average_day(self, max_altitude=10000):
+        """Compute the average wind profile for each avaliable hour of a day, over all
+        days in the dataset."""
+        altitude_list = np.linspace(self.elevation, max_altitude, 100)
         average_wind_profile_at_given_hour = {}
-        for hour in list(self.pressureLevelDataDict.values())[0].keys():
+        hours = list(self.pressureLevelDataDict.values())[0].keys()
+        for hour in hours:
             wind_speed_values_for_this_hour = []
             for dayDict in self.pressureLevelDataDict.values():
                 try:
                     wind_speed_values_for_this_hour += [
                         dayDict[hour]["windSpeed"](altitude_list)
                     ]
                 except KeyError:
-                    # Some day does not have data for the desired hour (probably the last one)
+                    # Some day does not have data for the desired hour
                     # No need to worry, just average over the other days
                     pass
-            average_wind_profile_at_given_hour[hour] = np.mean(
-                wind_speed_values_for_this_hour, axis=0
+            average_wind_profile_at_given_hour[hour] = [
+                np.mean(wind_speed_values_for_this_hour, axis=0),
+                altitude_list,
+            ]
+        self.average_wind_profile_at_given_hour = average_wind_profile_at_given_hour
+
+    # TODO: Create test
+    def plot_wind_profile_over_average_day(self, max_altitude=10000):
+        """Creates a grid of plots with the wind profile over the average day."""
+        # Check if needed data has already been computed
+        if self.average_wind_profile_at_given_hour is None:
+            self.process_wind_profile_over_average_day(max_altitude)
+
+        # Create grid of plots for each hour
+        hours = list(list(self.pressureLevelDataDict.values())[0].keys())
+        nrows, ncols = self._find_two_closest_integer_factors(len(hours))
+        fig = plt.figure(figsize=(ncols * 2, nrows * 2.2))
+        gs = fig.add_gridspec(nrows, ncols, hspace=0, wspace=0, left=0.12)
+        axs = gs.subplots(sharex=True, sharey=True)
+        for (i, j) in [(i, j) for i in range(nrows) for j in range(ncols)]:
+            hour = hours[i * ncols + j]
+            ax = axs[i, j]
+            ax.plot(*self.average_wind_profile_at_given_hour[hour], "r-")
+            ax.set_title(f"{float(hour):05.2f}".replace(".", ":"), y=0.8)
+            ax.set_xlim(0, max(self.average_wind_profile_at_given_hour[hour][0]))
```

Comment:
I've got a minor issue with this set_xlim. Idn how to better set the limits though.
I think one simple idea would be: `ax.set_xlim(0, max(self.average_wind_profile_at_given_hour[hour][0]) + 2)`

It can be important to visualizes wind shear phenomena, but there's no worry cause it's not going to happen to every time period.

![image](https://user-images.githubusercontent.com/63590233/170848219-e51f63c4-0730-44d9-877b-4f3a336aef43.png)